package.json

```diff
@@ -37,18 +37,20 @@
 		"demo": "http-server"
 	},
 	"dependencies": {
+		"@govflanders/vl-ui-accordion": "^3.12.17",
 		"vl-ui-accordion": "^3.0.11",
-		"vl-ui-core": "^7.1.1"
+		"vl-ui-core": "^7.2.2"
 	},
 	"peerDependencies": {
-		"vl-ui-accordion": "^3.0.10",
+		"@govflanders/vl-ui-accordion": "^3.12.17",
+		"vl-ui-accordion": "^3.0.11",
```

Comment:
is dat "logisch", dat info-tile een dependency heeft op accordion?
of is dat nu gewoon "een gemakkelijkheidsoplossing".

Reply:
Ja, de vl-info-tile maakt gebruik van de @govflanders accordion lib: `import '/node_modules/@govflanders/vl-ui-accordion/dist/js/accordion.js';` en de vl-ui-accordion CSS: `@import '/node_modules/vl-ui-accordion/dist/style.css';`.

Reply:
Ja van de onderliggende @govflanders accordion kan ik me nog voorstellen, maar de dependency op vl-accordion lijkt me niet "juist".

Reply:
Nochtans heeft de vl-info-tile toggleable variant nood aan de CSS van de vl-accordion. Een compromis zou kunnen zijn dat ik de volledige CSS van accordion hier opnieuw genereer. Dat impliceert een extra dependency op vl-toggle en vl-link.

Reply:
Ik heb die aanpassing gedaan, zorgt er dan ook voor dat de circulaire dependency verdwijnt.